index.html

```diff
@@ -27,3 +27,16 @@ <h3 class="panel-title"><a href="{{ post.url | prepend: site.baseurl }}">{{ post
     </div>
 {% endfor %}
 </ul>
+
+<div class="pagination"> {% if paginator.previous_page %}
+      {% if paginator.previous_page == 1 %}
+      <p class="prevLink"><a href="{{site.baseurl}}/" class="previous">newer entries</a></p>
```

Comment:
Perhaps "Newer Posts" is better. Is there any way to provide numeric pagination?